### ISSUE TITLE
Fix up prompts

### ISSUE THREAD

**Issue by rhom001** [rhom001/ec_library#6]
Current prompts for add() and modify() for Admin subclasses are a tad too basic or repetitive. If you have improved keywords, please comment down below.

**Comment by ghost** [comment]
you could make them like: add_user , add_series .... etc ...

**Comment by rhom001** [comment]
True, I could do that. Currently what I have is to reduce the amount of writing needed for prompts, I have a few simple prompts in the Format file.
